OpenRA.Game/Network/SyncReport.cs

```diff
@@ -68,17 +68,22 @@ void GenerateSyncReport(Report report)
 			report.Effects.Clear();
 
 			foreach (var actor in orderManager.World.ActorsHavingTrait<ISync>())
+			{
 				foreach (var syncHash in actor.SyncHashes)
-					if (syncHash.Hash != 0)
+				{
+					var hash = syncHash.Hash();
+					if (hash != 0)
```

Comment:
Style nit: please wrap this in braces to avoid the hanging multi-line braced block.

Reply:
Added.